src/add-remove.js

```diff
@@ -1,117 +0,0 @@
-import iconDelete from './delete.png';
-import iconDots from './icon_dots.png';
-import updateCheckbox from './update-chk.js';
-import './style.css';
-
-const addList = document.body.querySelector('.add-list');
-const inputList = document.body.querySelector('.input-list');
-const uList = document.body.querySelector('.ulist-child');
-
-const todoArray = [];
-
-export function removeTask(list) {
-  const index = Number(list.id);
-  list.remove();
-  todoArray.splice(index, 1);
-  todoArray.forEach((element, index) => {
-    element.index = index;
-  });
-  localStorage.setItem('data', JSON.stringify(todoArray));
-}
-
-export function editTask(event, list, checkboxImage, checkboxLabel) {
-  const index = Number(list.id);
-
-  if (event === 'label') {
-    checkboxImage.src = iconDelete;
-    checkboxImage.id = 'delete';
-    checkboxLabel.readOnly = false;
-    checkboxLabel.id = 'labelClick';
-    list.style.backgroundColor = 'yellow';
-    checkboxLabel.style.backgroundColor = 'yellow';
-  } else if (event === 'delete') {
-    removeTask(list);
-  } else if (event === 'labelClick') {
-    checkboxImage.src = iconDots;
-    checkboxImage.id = 'dots';
-    list.style.backgroundColor = 'white';
-    checkboxLabel.style.backgroundColor = 'white';
-    checkboxLabel.id = 'label';
-    checkboxLabel.readOnly = true;
-    todoArray[index].description = checkboxLabel.value;
-    localStorage.setItem('data', JSON.stringify(todoArray));
-  } else {
-    checkboxImage.src = iconDots;
-    checkboxImage.id = 'dots';
-    list.style.backgroundColor = 'white';
-    checkboxLabel.style.backgroundColor = 'white';
-    checkboxLabel.id = 'label';
-    checkboxLabel.readOnly = true;
-    todoArray[index].description = checkboxLabel.value;
-    localStorage.setItem('data', JSON.stringify(todoArray));
-  }
-}
-
-export function addTask(description, index, check) {
```

Comment:
- Good work so far, you made great efforts to make the Todo app.
- It would be great if you make use of the arrow function for clean code.